docs/mobile.md

```diff
@@ -0,0 +1,115 @@
+to use longuer timelocks delays in your configuration (`ChannelHandshakeConfig::our_to_self_delay`).
+
+With regards to handling in-flight HTLCs, a Lightning node not online when a HTLC has expired is
+taking the risk of seeing its counterparty canceling onchain the HTLC and thus loosing funds. 
+
+In case of sending a payment, if it doesn't succeed immediately (e.g waiting on the receiver to
+be online), the mobile should be back online later to remove the sent HTLC in cooperation with
+channel counterparty or worst-case scenario to broadcast onchain the commitment transaction and
+a corresponding HTLC-timeout. Setting a long CLTV delay at first hop relay offers a _time window_
+for how long the mobile device can be offline. Ideally, the Lightning application will send a
+notification to the user at half of the CLTV delay warning about the requirement to connect back
+to the network.
+
+On the UX-side, as soon the HTLC is sent on the channel, the user balance should be decreased
+accordingly as unlock secret might already be known by counterparty or receiver. In the future,
+the PTLC protocol extension will allow to acknowledge that the payment has been received by 
+expected payee before to decrease balance.
+
+In case of receiving a payment, if it doesn't succeed immediately (e.g waiting on payer interactions
+to receive the preimage), the mobile should be back online to hopefully learn payment preimage
+and thus either claim the received HTLC in cooperation with channel counterparty or worst-case scenario
+to broadcast onchain the commitment transaction and a corresponding HTLC-Success. Requiring a long
+CLTV delay at last hop offers a _time window_ for how long the mobile device can be offline. With
+LDK, this can be easily done by checking `event::PaymentReceived` at reception. Ideally, the 
+Lightning application will send a notification to the user at half of the CLTV delay warning about
+the requirement to connect back to the network.
+
+On the UX-side, the user balance should only be increased when a `revoke_and_ack` message is received
+from the counterparty or the HTLC has been claimed onchain. Otherwise, you can't exclude onchain
+claims races depriving the user from its payment.
+
+## Chain Validation
+
+A Lightning node needs access to the chain to verify channel utxo state. Failure to do so
+might lead to accept payments on a non-existent or already-closed channel, thus provoking a loss of
+funds. As chain validation is a costly processing, a mobile device might not have the bandwidth/CPU/
+storage resources to allocate for.
+
+In the Bitcoin ecosystem, mobile devices have been in practice relying on lightclient protocols
+such as Electrum, BIP37 or BIP157 to access chain data. If the server isn't operated by the mobile
+device user, it should be noted that funds security might be jeopardized by this entity.
+
+LDK offers a ready-to-integrate utility to fetch chain data from a Bitcoin Core node servicing
```

Comment:
I wouldn't really call it "ready-to-integrate" in the sense that we don't do any filtering in the code we have, only an API which allows you to build a filtering client. We *do* have a ready-to-integrate client that fetches full blocks from RPC or REST.

Reply:
Updated.